android/app/src/main/java/com/reactnativenavigation/react/NavigationReactGateway.java

```diff
@@ -90,6 +90,7 @@ public void onJsDevReload() {
 
     private ReactInstanceManager createReactInstanceManager() {
         ReactInstanceManager.Builder builder = ReactInstanceManager.builder()
+                .setUseOldBridge(true)
```

Comment:
Just confirmed with my colleague, he can't see any reason that it's still needed. Shall I remove it?